Core/clim-basic/regions.lisp

```diff
@@ -551,12 +551,13 @@
    (tr          :initarg :tr)))         
 
 (defmethod print-object ((ell elliptical-thing) stream)
-  (with-slots (start-angle end-angle tr) ell
-    (format stream "#<~A [~A ~A] ~A>"
-            (type-of ell)
-            (and start-angle (* (/ 180 pi) start-angle))
-            (and end-angle (* (/ 180 pi) end-angle))
-            tr)))
+  (print-unreadable-object (ell stream :type t :identity t)
+    (with-slots (start-angle end-angle tr) ell
+      (format stream "~A [~A ~A] ~A"
+              (type-of ell)
```

Comment:
I don't know if this has been mentioned before, but `print-unreadable-object … :type t` already prints the type so this shouldn't be needed.